### ISSUE TITLE
authToken: improve docs on required key scope 

### ISSUE THREAD

**Issue by JonasBa** [getsentry/sentry-webpack-plugin#243]
👋  Hey everyone, first of all thanks for the great work, Sentry is a joy to use!

**TLDR**; What is the minimum authToken access scope required for this plugin to function properly?

I have a slight question/proposal regarding authToken documentation in README.md as I have been currently adding this plugins to our build and wanted to minimize the scope of the authToken to the bare minimum (ideally to only allow it to upload the source maps). 

The current documentation states where the token can be obtained, but not what scope is required:
```
The authentication token to use for all communication with Sentry. Can be obtained from https://sentry.io/settings/account/api/auth-tokens/.
```

Besides that, the default UI for creating a new key seems to include a very broad scope of permissions (team:read, org:read, event:admin...). It seems that if a key like this is leaked, a potential attacker could get a set of very powerful permissions.

<img width="1163" alt="CleanShot 2020-12-05 at 10 39 41@2x" src="https://user-images.githubusercontent.com/9317857/101239224-36850f80-36e6-11eb-8e3a-a5c0f38fa038.png">

It would be great if just a few words could be added to the readme explaining what permissions are needed for the plugin to work, this would decrease the threat surface and could avoid some unpleasant security scenarios if someone obtains a leaked key!

Lmk if I missed something and thanks again!

**Comment by kamilogorek** [comment]
> 👋 Hey everyone, first of all thanks for the great work, Sentry is a joy to use!

Hey, thanks!

> TLDR; What is the minimum authToken access scope required for this plugin to function properly?

`project:releases` in itself is enough to do most of the tasks. The only additional permission, that is required _if_ you use `setCommits` option, is `org:read`.

Updated the docs https://github.com/getsentry/sentry-webpack-plugin/pull/244

Cheers! :)
